compiler/src/iree/compiler/Codegen/Common/test/strip_config_info.mlir

```diff
@@ -0,0 +1,103 @@
+// RUN: iree-opt --split-input-file --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(builtin.module(func.func(iree-codegen-strip-config-info)))))' %s | FileCheck %s
```

Comment:
Ah, now I think we should make this pass over `Operation` so that we don't have to manually specify the nesting. We can walk whatever the input op is and find functions ops first.